packages/typespec-python/HealthInsights/healthinsights.common/model.common.request.tsp

```diff
@@ -0,0 +1,125 @@
+import "./model.common.shared.tsp";
+
+using TypeSpec.Rest;
+
+namespace AzureHealthInsights;
+
```

Comment:
I think it is better to move it to test.

Reply:
oh i'm sorry going to get rid of it. I had it up for debug purposes and actually pushed it